client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/SslConfigurationBuilder.java

```diff
@@ -101,9 +101,6 @@ public SslConfigurationBuilder trustStorePassword(char[] trustStorePassword) {
    public void validate() {
       if (enabled) {
          if (sslContext == null) {
-            if (keyStoreFileName == null) {
-               throw log.noSSLKeyManagerConfiguration();
```

Comment:
looked to me strange to have gaps in log numbers :smile: . There's more log messages which seem to be unused, I removed them all in a additional commit (and created https://issues.jboss.org/browse/ISPN-6492)